web/src/components/dashboards/useFetchPersesDashboards.ts

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { useSafeFetch } from '../console/utils/safe-fetch-hook';
+import { useBoolean } from '../hooks/useBoolean';
+
+export const useFetchPersesDashboards = (): [[], boolean, string] => {
```

Comment:
I think it will be bette to abstract the Perses API in a `persesClient.ts` and then use this hook to consume the API. This will allow to extend it later easily. We should type the hook response so is easier to read.
```suggestion
export const usePerses = (): PersesHook => {
```

Reply:
I refactored this into `persesClient.ts` in the latest push.